roles/zooplus/tasks/teams.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Check if teams is installed
+  stat:
+    path: /usr/bin/teams
+  register: teams_existence
+
+- name: Download teams package
```

Comment:
There is a repository for teams. You should use the apt_repository module
https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html

Reply:
aded